frontend/javascripts/components/pricing_enforcers.tsx

```diff
@@ -75,14 +64,42 @@ export function PricingEnforcedButton({
   if (isFeatureAllowed) return <Button {...buttonProps}>{children}</Button>;
 
   return (
-    <Tooltip title={toolTipMessage} placement="right">
+    <Tooltip title={messages["organization.plan.feature_not_available"]} placement="right">
       <Button {...buttonProps} disabled>
         {children}
-        {showLockIcon ? <LockOutlined style={{ marginLeft: 5 }} /> : null}
+        <LockOutlined style={{ marginLeft: 5 }} />
       </Button>
     </Tooltip>
   );
-}
+};
+
+export const PricingEnforcedBlur: React.FunctionComponent<RequiredPricingProps> = ({
+  children,
+  requiredPricingPlan,
+}) => {
+  const currentPricingPlan = useSelector((state: OxalisState) =>
+    state.activeOrganization ? state.activeOrganization.pricingPlan : PricingPlanEnum.Basic,
+  );
+  const isFeatureAllowed = isPricingPlanGreaterEqualThan(currentPricingPlan, requiredPricingPlan);
+  if (isFeatureAllowed) return <>children</>;
```

Comment:
I think, this would render the string "children" :thinking: 

```suggestion
  if (isFeatureAllowed) return <>{children}</>;
```
or:
```
  if (isFeatureAllowed) return children;
```

Reply:
Ah, good catch. Unfortunately, `return children` does not work... well at least Typescript complains... (React actually was able to work with this nonetheless). Hence, I had to wrap it in a Fragment to satisfy the type checker.